rocketmq-broker/README.md

```
@@ -10,54 +10,178 @@ This module is mainly the implementation of the [Apache RocketMQ](https://github
-    -h, --help                    Print help
-    -V, --version                 Print version
-  ```
-
-Run the following command to start the name server
+```shell
+cargo run --bin rocketmq-broker-rust -- --help
+```
+
+**Output:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)